### PR TITLE
✅ Add links to complete and reject submissions

### PR DESCRIPTION
**Pull request by mermop**
https://trello.com/c/3pXFQ6FE/74-mark-submission-as-completed

Screen Shot 2020-01-28 at 11 06 23 AM

### CODE REVIEW

File: app/views/submissions/index.html.erb [hunk 1]
```diff
@@ -26,9 +27,10 @@
             <%= "⏸" unless submission.file.analyzed? %>
           </td>
           <td class="text-right">
-            <%= render partial: 'actions', locals: { submission: submission } %>
-            <%= link_to 'Delete', [@consultation, submission], method: :delete, class: 'btn btn-danger',
```
Comment (Rabid-Dan) [inline app/views/submissions/index.html.erb]: do we not allow submissions to be deleted anymore? or should this have gone into the edit_actions partial?

Comment (mermop) [reply app/views/submissions/index.html.erb]: I think we should probably make it harder to delete them at least, I can see people accidentally deleting the wrong thing at the moment

File: app/models/submission.rb [hunk 2]
```diff
@@ -35,7 +35,7 @@ class Submission < ApplicationRecord
     end
 
     event :complete_tagging do
-      transition started: :finished
+      transition started: :finished, if: :tagged?
```
Comment (Rabid-Dan) [inline app/models/submission.rb]: Is there a potential scenario where the user might want to mark a submission as completed without tagging it? I'm thinking something like an instance where they have 100 submissions to tag and get to number 25 then need to do something else. Maybe number 25 doesnt have anything that needs tagging but they want to mark it as complete so they know not to check it again when they resume tagging the other 75?

Comment (mermop) [reply app/models/submission.rb]: Fair enough, it's not too hard to reject it again

**Pull request merged**